model_signing/serialize_test.py

```diff
@@ -136,7 +115,7 @@ def test_file_chuncks(self):
 
     # File serialization raises error for negative chunk values.
     def test_file_negative_chuncks(self):
-        file = "model_file"
+        file = "v0_test_file_negative_chuncks"
```

Comment:
```suggestion
        file = "v0_test_file_negative_chunks"
```

We have this typo in several places